icons/ham.json

```diff
@@ -0,0 +1,20 @@
+{
+  "$schema": "../icon.schema.json",
+  "contributors": [
+    "karsa-mistmere"
```

Comment:
If you based this on another icon, you can inherit the contributors.

Reply:
Yes, it's based on my own changes, exactly. 😛